scripts/fetchdep.sh

```diff
@@ -17,10 +17,10 @@ clone() {
     fi
 }
 
-# Try the PR author's branch in case it exists on the deps as well.
-clone $TRAVIS_PULL_REQUEST_BRANCH
 # Try the target branch of the push or PR.
-clone $TRAVIS_BRANCH
+clone $BUILDKITE_BRANCH
+# Try the PR author's branch in case it exists on the deps as well.
+clone $BUILDKITE_PULL_REQUEST_BASE_BRANCH
```

Comment:
Thanks for bringing these back. The `clone` lines seem correct and are in the correct order, but the comments are mismatched (`$BUILDKITE_BRANCH` is the PR author's branch), so it we should swap the comments. I believe Riot Web's `fetch-develop.deps.sh` has the right ordering for both code and comments.